Dynatrace/dynatrace-operator
# Skip unnecessary requests to tenant within 15 min window

# Description

On dynakube reconciliation, request to the tenant are made that are unnecessary, as the data won't change that often. 
This PR adds a timeout, within these request will be skipped.

**Important:** For this improvement to work, the CRD has to be updated, as the lastAccess timestamps are new fields in the status object. - In case the CRDs are not updated, we skip back to the previous default, sending requests on each reconciliation.

## How can this be tested?

- Run multiple reconciliations and check if connections to the tenant are skipped.

## Checklist
